reg_form_ext/choices.py

```diff
@@ -0,0 +1,247 @@
+COUNTRY_CODES = [
```

Comment:
Yeah, for the sake of not maintaining this list, I [agree with using one of the django phonenumber libraries](https://github.com/open-craft/edx-custom-registration-form/pull/1#issuecomment-348395273).